server/src/main/java/org/elasticsearch/index/mapper/Mapping.java

```diff
@@ -18,14 +25,7 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import org.elasticsearch.Version;
-import org.elasticsearch.common.Strings;
-import org.elasticsearch.common.xcontent.ToXContent;
-import org.elasticsearch.common.xcontent.ToXContentFragment;
-import org.elasticsearch.common.xcontent.XContentBuilder;
-import org.elasticsearch.common.xcontent.XContentFactory;
-import org.elasticsearch.common.xcontent.XContentHelper;
-import org.elasticsearch.index.mapper.MapperService.MergeReason;
+import static java.util.Collections.unmodifiableMap;
```

Comment:
these should go back to normal , now correcting a previous wrong change made by myself :)